src/data/equiv/basic.lean

```diff
@@ -1198,6 +1199,14 @@ equiv.set.image_of_inj_on f s (λ x y hx hy hxy, H hxy)
 @[simp] theorem image_apply {α β} (f : α → β) (s : set α) (H : injective f) (a h) :
   set.image f s H ⟨a, h⟩ = ⟨f a, mem_image_of_mem _ h⟩ := rfl
 
+lemma equiv.set.image_symm_preimage {α β} {f : α → β} (hf : function.injective f) (u s : set α) :
```

Comment:
Aren't you already in the `equiv.set` namespace here?

Reply:
yes, you're right